docs/tutorials/tut-console-CR-security-guidelines.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Security guidelines for MicroFrontend and ClusterMicroFrontend CRs
-type: Configuration
+type: obsolete?
 ---
```

Comment:
Just a note here so you didn't forget to remove/update this one and the similar ones ;)